cartridge/admin.lua

```diff
@@ -105,6 +105,11 @@ local function get_server_info(members, uuid, uri)
         ret.clock_delta = member.clock_delta * 1e-6
     end
 
+    if ret.status == 'healthy' and member.payload.replication_status == 'broken' then
```

Comment:
Please, consider this https://github.com/tarantool/tarantool/commit/f2cbc886970451628b335b9b8216be25a0e54c3a#diff-faad149697113fad79bb54aeb755b0b5R4